airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql

```diff
@@ -8,7 +8,7 @@ description: |
 fields:
   calitp_itp_id: Feed ITP ID.
   calitp_url_number: Feed URL number.
-  textPayload: Error message that contains feed key, header length, download url, and error response.
+  token_cleaned_textPayload: Error message that contains feed key, header length, download url, and error response. API Tokens are replaced in the string with API_TOKEN
```

Comment:
Let's try to be consistent with what kind of case is being used.

Reply:
in the gtfs_archiver it is "textPayload".. so would you prefer "textpayload" or "text_payload"

Reply:
changed it to text_payload